.golangci.yml

```diff
@@ -172,6 +172,10 @@ issues:
       linters:
         - funlen
       text: "Function 'TestMonitor' is too long"
+    - path: pkg/tools/opa/tokens_valid_policy.go
```

Comment:
Looks like there's a conflict in this file that needs resolving.

Tip on gosec

You can use the comment
```go
 // #nosec
```
to mark lines for gosec to ignore, and thus avoid having to change the .golangci.yml file :)

Reply:
thanks! Fixed.